### ISSUE TITLE
[CoE Starter Kit - QUESTION] Flow Orphan Status Clarification

### ISSUE THREAD

**Issue by a-wiesinger** [microsoft/coe-starter-kit#6763]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

I'm at the 2023 Power Platform Conference (woot!) and I asked a question of Marc regarding the CoE Starter Kit and he asked me to post my question here :) 

Our Starter Kit is mostly working as expected. However, I have a group of flows that lost their maker, since the person left the company, and their "owner" field went blank; these flows have co-owners assigned to them. They are now showing orphaned status and I'm having a bit of a hard time trying to determine what should officially be driving this.

First, I would expect that if a flow has co-owners, the flow would not be flagged as orphaned. Since these were non-solution flows and I couldn't edit the owner, I went in and added these orphaned flows to a solution so I could edit that and set the proper person in there. However, after doing this, I'm still seeing these flows show in our list of orphaned flows in the Power Platform Admin View app. I don't know if this is because the admin flow that checks for orphaned status has not yet run, or if it's because the maker has left and it doesn't matter what I do to this thing, it will always show orphaned. In the Power Platform Admin View app when I select the flow and hover over the "Flow is Orphaned" category, the tooltip that comes up says very explicitly that the Flow is Orphaned status is not set by the **maker** leaving the company:

<img width="621" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/70759917/f2651c35-068d-4c0b-848d-2abf13daa866">

I guess my question is a two-parter:
1. What **exactly** is the driver that says a flow is orphaned? Is it that the maker is gone, or is it that it doesn't have an owner set properly (even though these had 2 co-owners)?
2. What is the admin flow in the CoE Starter Kit that goes out to check for orphaned status? The only flow I could find that seemed to be related is the "CLEANUP - Admin | Sync Template v3 (Orphaned Makers)" one. And the naming on this leads me to believe that it **doesn't** rely on owner, but looks for maker instead.

Thank you for taking a look and I look forward to any assistance on this that can be provided. 

### What solution are you experiencing the issue with?

None

### What solution version are you using?

June 2023

### What app or flow are you having the issue with?

CLEANUP - Admin | Sync Template v3 (Orphaned Makers)

### What method are you using to get inventory and telemetry?

Cloud flows

AB#729

**Comment by Jenefer-Monroe** [comment]
Hello. Yes we have been unable to get access to the flow owner information from the product team. We are re-doing this work as a whole once we have one more change expected from the product team. 

We likely will not be auto-assigning a co-owner but we hope to offer them to the admin as suggestions instead. 

Anyway, I'll assign this to that feature so we can use your write up as a suggestion once we are unblocked and able to do that work.

**Comment by a-wiesinger** [comment]
Thank you so much! At least I know that I was understanding the intention of the orphaned flows status correctly. Looking forward to seeing future implementations where we have access to owner information. :)

**Comment by dtheilen** [comment]
I'd like to add to this. My customer has been updating the owner of solution-aware flows that became orphaned. While there's no other co-owners on the flow, there is now an active primary owner. CoE is still treating it as an orphan object (gov version 3.12). Will future releases enable a way to flag something as NOT orphaned, via the Admin View app perhaps?

**Comment by larhem** [comment]
IS there any update on this issue? We still see a lot of orphaned flows even if we have changed owner to a valid user after maker of the flow has left the company.

**Comment by AmarSaiRam** [comment]
We are consolidating multiple related enhancement requests into a single comprehensive issue to streamline tracking and implementation. This ticket is now closed because its scope is fully covered under the new consolidated issue focused on centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: FEATURE TITLE Enhancement: Centralized Management for Orphaned Compone…](https://github.com/microsoft/coe-starter-kit/issues/10319)

Thank you for your valuable input and collaboration
